Document/0x05c-Reverse-Engineering-and-Tampering.md

```diff
@@ -582,7 +582,7 @@ main[1] locals
 flag = false
 ```
 
-You've now reached a call to `setCancelable` with the argument `false`. Set the variable to true with the `set` command and resume.
+You've now reached a call to `setCancelable` with the argument `false`. Set the variable to "true" with the `set` command and resume.
```

Comment:
```suggestion
You've now reached a call to `setCancelable` with the argument `false`. Set the variable to `true` with the `set` command and resume.
```

Can you doublecheck all true/false values and make them `true` or `false` instead of "true" or "false" ?